UPGRADE_GUIDE.md

```diff
@@ -1,5 +1,80 @@
 # Upgrade Guide
 
+## From `0.2.6` to `0.2.7`
+
+### Worker SDK `14.0.1` upgrade
+
+There are a number of breaking changes to contend with for the Worker SDK upgrade:
+
+#### Removal of `Vector3f` and `Vector3d`
+
+These two schema types are no longer available in the schema standard library. You can replace their definitions by defining a schema file in your project:
```

Comment:
```suggestion
These two schema types are no longer available in the standard schema library. You can replace their definitions by first defining a schema file in your project:
```

it's what we call it in spos docs 🤷‍♂ 
https://docs.improbable.io/reference/14.0/shared/schema/standard-schema-library